.travis.yml

```diff
@@ -14,17 +14,23 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-language: minimal
+os: linux
+dist: focal
+language: shell
```

Comment:
Why are you changing this to `language: shell` over `language: minimal`?

Reply:
Fix Travis CI build config validation issues
![Screenshot 2020-07-06 at 20 42 48](https://user-images.githubusercontent.com/3709715/86627993-7de96180-bfc9-11ea-8da0-a739ce34138b.png)